train_diffusion_prior.py

```diff
@@ -154,10 +157,10 @@ def train(image_embed_dim,
         os.makedirs(save_path)
 
     # Get image and text embeddings from the servers
-    print_ribbon("Downloading embeddings - image and text")
-    image_reader = EmbeddingReader(embeddings_folder=image_embed_url, file_format="npy")
-    text_reader  = EmbeddingReader(embeddings_folder=text_embed_url, file_format="npy")
-    num_data_points = text_reader.count
+    print_ribbon("Downloading Image Embeddings")
+    image_reader = EmbeddingReader(embeddings_folder=image_embed_url,
+                                   metadata_folder=meta_url, meta_columns=['caption'], file_format="parquet_npy")
+    num_data_points = image_reader.count
```

Comment:
@rom1504 This is the biggest change, moving to a single EmbeddingReader. 

Given how much better Katherine's results are with text-conditioning I feel as though it is a safe bet to drop support for the embedding-only training scheme we had previously.